### ISSUE TITLE
Split data into train, validation, and test set

### ISSUE THREAD

**Issue by ANGELLOPARR** [ANGELLOPARR/csc466-project#11]
We're gonna need the data split into 3 partitions for training the model, monitoring overfitting, and getting our final evaluations. This split should be 70% / 15% / 15% training, validation, and testing. Also, the sets should have a roughly even distribution of each type of class.

Definition of done: 3 different datasets for training, validation, and testing data that have a roughly equal amount of output classes
